### PR TITLE
hw01 - Simon Yamawaki Shachter

### PR DESCRIPTION
**Pull request by simonys**
I already had R installed on my computer, but had some difficulty connecting with github as I had used github for some past projects and assignments and all my configurations were customized for those purposes under a different username and account. So after just under 15 minutes of struggles for each problem I was able to successfully connect and commit to git on my machine and github online.

Using the [Github Markdown Guide](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) I was able to add in different text, images, and play around with the different formatting. I am not a design person and it was a little hard to make things look normal (I hope you think they do) and I couldn't decide on a lot of fairly minor choices, but in the end didn't want to make anything too complicated while trying to use most of the functionality of the markdown language.

One problem I had was trying to resize images. My original picture appeared quite large, and I wanted to be a little less of an egoist, so I hit the message boards to try to figure out how to resize images, but without luck. Then I learned that you can also just use html code in markdown to produce what you want, so I feel like I slightly cheated by using html rather than markdown to insert my photo, but it was a good learning process nonetheless and ultimately successful on the front-end.

### CODE REVIEW

**Comment by jeinpark** [comment]
This is a very successful homework assignment. Not only did it give a thorough biography of the student, but Simon also seamlessly implemented Markdown text styles and links. They were successfully transferred into the Github view as well. I cannot find any issues or complications - great job!

**Comment by yeseulbyeon** [comment]
Coding Style: Excellent
Coding Strategy: Excellent
Presentation: Excellent
Achievement, creativity: Satisfactory
Ease of Access: Excellent

I agree with Jein -- this was a great bio. Your coding style is tidy, and I especially liked your use of links, both embedded inside the paragraph and also in the form of a bulleted list at the bottom of the page.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
 # hw01
 Repository for homework 1
+
+## Simon Yamawaki Shachter
+### Autobiography
+
+<img src="https://pbs.twimg.com/profile_images/575678523521683458/Psv53-eU_400x400.jpeg" width="200">
```
Comment (nnickels) [inline README.md:7]: Good work with html;  would work, so long as your jpg is saved in the local repository and pushed back to Github as well.

**Review by nnickels — COMMENTED** [review]
Evaluation HW-01 for Simon Schachter
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |          |       X       |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |           |       X       |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Nice job on homework one! You display some great markdown aspects and produce a good looking README
* It's tricky to get everything working sometimes when you're not starting fresh and have used R and Git before, so great work getting everything up and running.
* You may have better luck with resizing images if you call an image saved in your local repo with its file path, then append the image using " =WIDTHxHEIGHT"

**Pull request closed without merging**